php/commands/term.php

```diff
@@ -417,6 +417,36 @@ public function url( $args ) {
 		}
 	}
 
+	/**
+	 * Recount the term_taxonomy count for one or more taxonomies.
+	 *
+	 * <taxonomy>...
+	 * : One or more taxonomies to update
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp term recount category post_tag
+	 *
+	 *     wp taxonomy list --field=name | xargs wp term recount
+	 */
+	public function recount( $args ) {
+		foreach( $args as $taxonomy ) {
+
+			if ( ! taxonomy_exists( $taxonomy ) ) {
+				WP_CLI::warning( sprintf( "Taxonomy %s does not exist.", $taxonomy ) );
+			} else {
+
+				$terms = get_terms( $taxonomy, array( 'hide_empty' => false ) );
```

Comment:
It might be helpful to have a parameter to specify the term ID or slug (or a list of these) to update specific terms, though it gets tricky if we try to do that for multiple taxonomies.  I'm just not sure of a scenario where someone wouldn't want to update the count for one term while leaving other terms with possibly incorrect counts.